composer.json

```diff
@@ -246,6 +245,7 @@
             "GPBMetadata\\Google\\Cloud\\Apigeeregistry\\": "ApigeeRegistry/metadata",
             "GPBMetadata\\Google\\Cloud\\Asset\\": "Asset/metadata",
             "GPBMetadata\\Google\\Cloud\\Assuredworkloads\\": "AssuredWorkloads/metadata",
+            "GPBMetadata\\Google\\Cloud\\Audit\\": "CommonProtos/metadata/Audit",
```

Comment:
All other references end with `*/metadata` except the ones in this PR, is it intentional?

Reply:
It is! It's because we have generated classes from 3 different protos (in 3 different namespaces).